### PR TITLE
Use mc functional tests for verifying build

### PR DESCRIPTION
**Pull request by balamurugana**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: buildscripts/verify-build.sh [hunk 1]
```diff
@@ -15,56 +15,35 @@
...
-}
+set -e
+set -E
+set -o pipefail
 
-function assert_run_mc_command()
-{
-    outfile="$WORK_DIR/cmd.out.$RANDOM"
+if [ ! -x "$PWD/minio" ]; then
+    echo "minio excutable binary not found in current directory"
```
Comment (poornas) [inline buildscripts/verify-build.sh]: typo here - should be executable

**Review by poornas — CHANGES_REQUESTED** [review]
Couldn't test this script as the reference to https://raw.githubusercontent.com/minio/mc/master/functional-tests.sh will be resolved only after the minio/mc PR #2253 to add functional tests  is merged.

File: buildscripts/verify-build.sh (hunk 1 above)
Comment (balamurugana) [reply buildscripts/verify-build.sh]: I am trying to say `minio binary as executable` not found.  Is it good idea to say `minio binary executable not found ...`?

Comment (poornas) [reply buildscripts/verify-build.sh]: s/excutable/executable/

Comment (balamurugana) [reply buildscripts/verify-build.sh]: got it

**Comment by balamurugana** [comment]
@poornas This PR is testable now.

**Review by poornas — CHANGES_REQUESTED** [review]
functional-tests.sh needs to be made executable. I get a permission denied.

**Comment by balamurugana** [comment]
> functional-tests.sh needs to be made executable. I get a permission denied.

You could need to use the latest refresh of this PR

**Review by poornas — APPROVED** [review]
Looks good.

**Review by kannappanr — APPROVED** [review]
Verified the changes. LGTM.

**Pull request merged**